config/seata-config-consul/src/main/java/io/seata/config/consul/ConsulConfiguration.java

```diff
@@ -53,10 +53,9 @@
     private static final String FILE_CONFIG_KEY_PREFIX = FILE_ROOT_CONFIG + FILE_CONFIG_SPLIT_CHAR + CONFIG_TYPE + FILE_CONFIG_SPLIT_CHAR;
     private static final int THREAD_POOL_NUM = 1;
     private static final int MAP_INITIAL_CAPACITY = 8;
-    private static ExecutorService consulConfigExecutor = null;
-    private static ExecutorService consulNotifierExecutor = null;
```

Comment:
why  io.seata.config.etcd3.EtcdConfiguration#addConfigListener use `etcdNotifierExecutor`, but here don't use `consulNotifierExecutor`

Reply:
In fact, their roles are the same, but due to the different implementation time, the logic of my thinking may be different, leading to different variable names.also,that's why there were two ExecutorService,which the "ExecutorService consulNotifierExecutor" is useless.

Reply:
But now I've decided to change the name to "consulNotifierExecutor" so that they can be better understood.